lib/game.rb

```diff
@@ -0,0 +1,97 @@
+class Game
+
+  attr_accessor :player_1, :player_2, :board, :token
+
+  #defines a constant WIN_COMBINATIONS with arrays for each win combination
+  WIN_COMBINATIONS = [
+  [0,1,2],
+  [3,4,5],
+  [6,7,8],
+  [0,3,6],
+  [1,4,7],
+  [2,5,8],
+  [0,4,8],
+  [2,4,6]
+  ]
+
+  def initialize(player_1=Players::Human.new("X"),player_2=Players::Human.new("O"),board=Board.new)
+    #defaults to two human players, X and O, with an empty board
+    #@"board" contains an instance of a board
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player #returns the correct player, X, for the third move
+      #if sum of x's and o's is even return player_1, else return player_2
```

Comment:
Consider using the `turn_count` method from the board class. 

`@board.turn_count.even? ? @player_1 : @player_2`